code/modules/hydroponics/grown/flowers.dm

```diff
@@ -164,7 +164,7 @@
 
 /obj/item/grown/sunflower/attack(mob/M, mob/user)
 	to_chat(M, "<font color='green'><b> [user] smacks you with a sunflower!</font><font color='yellow'><b>FLOWER POWER<b></font>")
```

Comment:
Not actually one of your modifications, but you might want to fix these bold tags because they're all over the place.